### ISSUE TITLE
Two minor reversions

### ISSUE THREAD

**Issue by CrossEye** [ramda/ramda.github.io#265]
Putting an issue so we won't forget to deal with this.  Somehow we lost the change we did to switch the URL shortener from Google to Tinyurl.  Also, I'm 95% certain that the title on the REPL page used to be "Ramda REPL".  Now it reads "Ramda Documentation."

I'm on other things now, but if no one beats me to this, I will come back and look sometime soon.

**Comment by customcommander** [comment]
There is still an open issue to replace the Google URL shortener #220.

**Comment by CrossEye** [comment]
Yes, but it was working in the previous version.  I don't recall how or when that was fixed.  Not a big deal, as we lived without it for a while.  I'm briefly back from several days away, and won't get much done today, but should be back on it tomorrow.
